<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHM_Serveur</class>
 <widget class="QWidget" name="IHM_Serveur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>724</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>724</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IHM_Serveur</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port d'écoute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPort">
       <property name="minimum">
        <number>5555</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLancerServeur">
       <property name="text">
        <string>Lancer le serveur</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditEtat"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetReservations">
     <property name="maximumSize">
      <size>
       <width>732</width>
       <height>208</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>idResa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Reference</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vol</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Place</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prénom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonQuitter">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>IHM_Serveur</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
